app/Http/Controllers/Api/PostApiController.php

```diff
@@ -3,6 +3,7 @@
 namespace App\Http\Controllers\Api;
 
 use App\Http\Controllers\Controller;
+use App\Http\Requests\PostRequest;
```

Comment:
This is okay for now since the creation and update request have the same validations and functionalities. But in actual projects, do keep in mind to use a proper naming convention:
Eg. PostUpdateRequest. PostStoreRequest